menas/src/main/scala/za/co/absa/enceladus/menas/controllers/DatasetController.scala

```diff
@@ -44,6 +46,13 @@ class DatasetController @Autowired()(datasetService: DatasetService)
 
   import scala.concurrent.ExecutionContext.Implicits.global
 
+  @GetMapping(Array("/latest"))
+  @ResponseStatus(HttpStatus.OK)
+  def getAll(@RequestParam(value = "missing_property", required = false) missingProperty: Optional[String]): CompletableFuture[Seq[MenasReference]] = {
```

Comment:
This is the first time I am ever hearing about `MenasReference` 👀 
Do we know what is that collection field? As it is always empty.

Reply:
I decided to reuse an existing object, so `MenasReference` made sense cause I saw it used in  `UsedIn` although sometimes it's used for other purposes too. Probably it would make sense to set collection as "Dataset"